SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -19,12 +19,17 @@ DEFINE_LOG_CATEGORY(LogSpatialWorkerConnection);
 
 namespace
 {
-SpatialGDK::ComponentData ToComponentData(FWorkerComponentData* Data)
+SpatialGDK::ComponentData ToComponentData(const FWorkerComponentData* Data)
```

Comment:
Can we avoid littering these kinds of helper in local files? They belong to some generally available utility header.